filters/builtin/comment.go

```diff
@@ -0,0 +1,50 @@
+package builtin
+
+import (
+	"github.com/zalando/skipper/eskip"
+	"github.com/zalando/skipper/filters"
+)
+
+type comment struct{}
+
+// NewComment is a filter to comment a filter chain. It does nothing
+func NewComment() filters.Spec {
+	return &comment{}
+}
+
+func (*comment) Name() string {
+	return filters.CommentName
+}
+
+func (c *comment) CreateFilter(args []interface{}) (filters.Filter, error) {
+	return c, nil
+}
+
+func (*comment) Request(filters.FilterContext) {}
+
+func (*comment) Response(filters.FilterContext) {}
+
+type CommentPreProcessor struct{}
+
+func (cpp *CommentPreProcessor) Do(routes []*eskip.Route) []*eskip.Route {
+	for _, r := range routes {
+		toDelete := make(map[int]struct{})
+
+		for i, f := range r.Filters {
+			if f.Name == filters.CommentName {
+				toDelete[i] = struct{}{}
+			}
+		}
```

Comment:
Why not
```go
for _, r := range routes {
	var ff []eskip.Filter
	for _, f := range r.Filters {
		if f.Name != filters.CommentName {
			ff = append(ff, f)
		}
	}
	r.Filters = ff
}
```

Reply:
So because we want to save a filter metric we will copy all filters :)

Reply:
done